network-api/networkapi/nav/templatetags/nav_tags.py

```diff
@@ -28,7 +28,7 @@ def get_dropdown_id(**kwargs):
 
 
 @register.simple_tag(takes_context=True)
-def check_if_dropdown_is_active(context, dropdown_id):
+def check_if_dropdown_can_be_active(context, dropdown_id):
```

Comment:
This is just a simple rename. The function checks if a dropdown contains a link that points to the current page. This means if two dropdowns have the same link, this will return True for both cases and the front end will end up showing duplicate highlight on both dropdown titles. I renamed function so it's less confusing since now we are using JS to determine out of all the "eligible" dropdowns, which one should gets the highlight.